packages/selenium/test/RunPartial/TestBase.cs

```diff
@@ -0,0 +1,166 @@
+            ";axe.runPartial = () => { throw new Error(\"No runPartial\")}";
+
+        [OneTimeSetUp]
+        public void OneTimeSetup()
+        {
+            var assembly = typeof(TestBase).GetTypeInfo().Assembly;
+            axeSource = new BundledAxeScriptProvider().GetScript();
+
+            axeCrashScript = File.ReadAllText(
+                Path.Combine(TestContext.CurrentContext.TestDirectory, "fixtures/axe-crasher.js")
+            );
+
+            dylangConfigPath = FixturePath("dylang-config.json");
+
+            driver = NewDriver();
+        }
+
+        [OneTimeTearDown]
+        public void TearDown()
+        {
+            driver.Close();
+            driver.Quit();
+        }
+
+        protected AxeBuilderOptions CustomSource(string axeSource)
+        {
+            return new AxeBuilderOptions
+            {
+                ScriptProvider = new StringAxeScriptProvider(axeSource)
+            };
+        }
+        protected void GoToResource(string url)
+        {
+            driver.Navigate().GoToUrl(ResourceUrl(url));
+        }
+
+        protected void GoToFixture(string url)
+        {
+            driver.Navigate().GoToUrl(FixtureUrl(url));
+        }
+
+        protected void GoToUrl(string url)
+        {
+            driver.Navigate().GoToUrl(url);
+        }
+
+        protected void AssertHasViolations(AxeResult results, params string[] rules)
+        {
+            var violations = results.Violations;
+            foreach (var violation in violations)
+            {
+                Console.WriteLine($">> {violation.Id}");
+            }
+
+            Assert.That(violations.Length, Is.EqualTo(rules.Length));
+
+            foreach (var rule in rules)
+            {
+                var found = false;
+                foreach (var vio in violations)
+                {
+                    if (vio.Id == rule)
+                    {
+                        found = true;
+                        break;
+                    }
+                }
+
+                if (!found)
+                {
+                    Console.Error.WriteLine($"Expected '{rule}' violation but could not find.");
+                    Assert.IsTrue(found);
+                }
+            }
+        }
+
+        private static void EnsureWebdriverPathInitialized(ref string driverPath, string dirEnvVar, string binaryName, IDriverConfig driverManagerConfig)
+        {
+            LazyInitializer.EnsureInitialized(ref driverPath, () =>
+            {
+                var dirFromEnv = Environment.GetEnvironmentVariable(dirEnvVar);
+                if (dirFromEnv != null)
+                {
+                    return $"{dirFromEnv}/${binaryName}";
+                }
+                else
+                {
+                    return new DriverManager().SetUpDriver(driverManagerConfig);
+                }
+            });
+        }
+        public static IWebDriver NewDriver()

```

Comment:
It would be nice for these tests to be consistent with (most of) the existing integration tests and run against firefox as well as chrome.